core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -1187,6 +1192,16 @@ angular.module('oppia').directive('conversationSkin', [
             $scope.showPendingCard();
           };
 
+          $scope.clickContinueButtonFn = function(interactionId) {
+            if (interactionId === 'CONTINUE') {
```

Comment:
Are you sure that this is the right interactionId? Have you tested end-to-end?

I believe it is supposed to be 'Continue' -- but in that case, why wasn't this discovered when testing manually?